client/src/components/Visualization/charts/Geochart.js

```diff
@@ -4,45 +4,70 @@
  * Date: 2020/07/13
  */
 
-import { Chart } from 'react-google-charts';
-import React from 'react';
 import { ads } from '../../../firebase/FirestoreDocumentReader';
+import { Chart } from "react-google-charts";
 import firebase from '../../../firebase/firebase';
+import React, { useState, useEffect } from 'react';
 import { states } from './StateDataParser';
+import { app, database } from '../../../firebase/firebase';
 
 const Geochart = () => {
-  /*
-   * Currently, getData assigns a meaningless random number to each state.
-   * In the future, once the states dictionary contains ad information, getData will
-   *    retrieve information such as minimum ad impressions and maximum ad spend.
-   */
-
-  function getData() {
-    const data = [['State', 'Random Number']];
-    for (const state in states) {
-      data.push([state, Math.floor(Math.random() * Math.floor(1000))]);
+
+  const [adTotal, setAdTotal] = useState(0);
+  const width = "700px";
```

Comment:
nit: if you just put the number 700, the component will automatically convert that to `700px` :D